src/main/java/jp/ikedam/jenkins/plugins/groovy_label_assignment/GroovyLabelAssignmentAction.java

```diff
@@ -102,6 +126,15 @@ public Label getAssignedLabel(SubTask task)
      */
     public Label getAssignedLabel()
     {
-        return label;
+        return Jenkins.getInstance().getLabel(getLabelString());

```

Comment:
NIT (?): Jenkins.getInstance() may return null. Realistic case: somebody uses this method in the new readResolve implementation. I'm not sure what happens during the queue loading